.github/workflows/ost.yml

```diff
@@ -0,0 +1,10 @@
+name: OST
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  ost:
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/ost'}}
```

Comment:
ideally add check for approved PRs - https://github.com/oVirt/ovirt-system-tests/blob/master/.github/example-trigger.yaml#L9

Reply:
and maybe some feedback when it is not approved, perhaps as a separate job with opposite "if" condition. If you create something better please contribute to https://github.com/oVirt/ovirt-system-tests/blob/master/.github/example-trigger.yaml (via gerrit still)